x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/configure_pipeline.tsx

```diff
@@ -62,6 +67,22 @@ export const ConfigurePipeline: React.FC = () => {
   const nameError = formErrors.pipelineName !== undefined && pipelineName.length > 0;
   const emptySourceFields = (sourceFields?.length ?? 0) === 0;
 
+  const modelOptions: Array<EuiSuperSelectOption<string>> = [
+    {
+      disabled: true,
+      inputDisplay: i18n.translate(
+        'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.configure.model.placeholder',
+        { defaultMessage: 'Select a model' }
+      ),
+      value: MODEL_SELECT_PLACEHOLDER_VALUE,
+    },
+    ...models.map((model) => ({
+      dropdownDisplay: <MlModelSelectOption model={model} />,
+      inputDisplay: model.model_id,
+      value: model.model_id,
```

Comment:
Is it possible to have `model_id = ''`, if yes what would happen?

Reply:
I don't think so currently, we get models from the ml api which only returns models in the space, and all of those should ids.

But let me double check that. If the api does possible return models with empty `model_id` it would of been an issue before this PR.

Reply:
Double checked and the API we currently use omits the entire model if it's not in the Space, so `model_id` will never be `''`